modules/vaos/spec/services/middleware/vaos_logging_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../../support/fixture_helper'
+
+describe VAOS::Middleware::VaosLogging do
```

Comment:
Could we add a spec that confirms no PII is logged?

Reply:
how? And what are you characterizing as PII? Because we definitely need to log all of the following:

ICN, JTI, the URL (which likely also include ICN or EDIPI), the users UUID, etc.

Reply:
Yep those are fine, but I'm pretty sure the one that comes back from VAMF includes SSN

Reply:
oh, well we're only logging the JTI from the decoded header, so if it includes anything else it shouldn't appear in logs.

Reply:
We are not logging the entire decoded JWT. Only the JTI key.